lms/static/js/instructor_dashboard/membership.js

```diff
@@ -463,13 +465,23 @@ such that the value can be defined later than this assignment (file load order).
             if (successes.length && dataFromServer.action === 'add') {
                 // Translators: A list of users appears after this sentence;
                 renderList(gettext('These users were successfully added as beta testers:'), (function() {
-                    var j, len1, results;
-                    results = [];
+                    var j, len1, activeUsers, inActiveUsers;
+                    activeUsers = [];
+                    inActiveUsers = [];
                     for (j = 0, len1 = successes.length; j < len1; j++) {
                         sr = successes[j];
-                        results.push(sr.identifier);
+                        if (sr.is_active) {
+                            activeUsers.push(sr.identifier);
+                        } else {
+                            inActiveUsers.push(sr.identifier);
+                        }
+                    }
+                    if (inActiveUsers.length) {
+                        renderList(message, inActiveUsers);
+                    }
+                    if (activeUsers.length) {
+                        return activeUsers;
```

Comment:
I feel like it would make sense to compile the list of `inactiveUsers` and `activeUsers` outside of the renderList function, and then just call `renderList` twice? Like:

```javascript
renderList(gettext('These users were successfully added as beta testers:'), activeUsers));
renderList(gettext('These users could not be added as beta testers because their accounts are not yet activated:'), inactiveUsers));
```